### PR TITLE
rename: change package names to kalix

### PR DESCRIPTION
**Pull request by jrudolph**
Trying to keep compatibility with existing proxy for now to make tests pass.

Refs: [#1289](https://github.com/lightbend/akkaserverless-framework/issues/1289)

### CODE REVIEW

**Comment by jrudolph** [comment]
Hmm, one sample fails with

```
Could not pull image: error pulling image configuration: download failed after attempts=1: error parsing HTTP 400 response body: invalid character '<' looking for beginning of value: "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<Error><Code>ExpiredToken</Code><Message>The provided token has expired.</Message>
```

File: codegen/core/src/test/scala/com/lightbend/akkasls/codegen/ExampleSuiteBase.scala [hunk 1]
```diff
@@ -32,12 +32,13 @@ abstract class ExampleSuiteBase extends munit.FunSuite {
 
   def propertyPath: String
 
-  def regenerate(testDir: java.io.File): Boolean =
+  def regenerate(testDir: java.io.File): Boolean = regenerateAll || {
```
Comment (ennru) [inline codegen/core/src/test/scala/com/lightbend/akkasls/codegen/ExampleSuiteBase.scala:35]: This is a change in logic, but seems OK.

File: codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/Types.scala [hunk 2]
```diff
@@ -1,74 +0,0 @@
-/*
```
Comment (ennru) [inline codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/Types.scala:1]: This file changed too much, but still exists: https://github.com/lightbend/akkaserverless-java-sdk/pull/857/files#diff-0c057024dc21c735eec15f1d48e8051f473cbbf76c986a5ac0e0ba27477f07ac

**Review by ennru — APPROVED** [review]
LGTM.

File: sdk/java-sdk/src/main/java/kalix/javasdk/package-info.java [hunk 3]
```diff
@@ -17,25 +17,25 @@
  *   <tbody>
  *     <tr class="altColor">
  *       <th class="colFirst" scope="row">Action</th>
- *       <td class="colLast"><a href="{@docRoot}/com/akkaserverless/javasdk/action/package-summary.html"><code>com.akkaserverless.javasdk.action</code></a></td>
+ *       <td class="colLast"><a href="{@docRoot}/com/akkaserverless/javasdk/action/package-summary.html"><code>kalix.javasdk.action</code></a></td>
```
Comment (jrudolph) [inline sdk/java-sdk/src/main/java/kalix/javasdk/package-info.java:20]: These links will have to be fixed. But I wasn't going for completeness here, yet.

File: codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/Types.scala (hunk 2 above)
Comment (jrudolph) [reply codegen/scala-gen/src/main/scala/com/akkaserverless/codegen/scalasdk/impl/Types.scala:1]: Yep, `git blame` seems to understand the move (though most of the lines still have changed...).

**Pull request merged**